uint32 can_id
uint32 data_high  # 앞 4바이트 (빅엔디언)
uint32 data_low   # 뒤 4바이트 (빅엔디언)
